extensions/aligning_guidelines/README.MD

```diff
@@ -20,3 +24,72 @@ const deactivate = initAligningGuidelines(myCanvas, options);
 
 deactivate();
 ```
+
+### custom function
+
+```ts
+import { initAligningGuidelines } from 'fabric/extensions';
+import { FabricObject } from 'fabric';
+
+// You can customize the return graphic, and the example will only compare it with sibling elements
+initAligningGuidelines(myCanvas, {
+  getObjectsByTarget: function (target) {
```

Comment:
I tried this PR and tried using this function to filter out objects that have properties `isGridLine` on them
like `if (!object.isGridLine) { set.add(object) }`
but then no guidelines happened at all
was I using it wrong maybe?

Reply:
@Smrtnyk 
Can you provide a simple demo to reproduce your issue? This will help me locate the problem. Regarding this function, if customized, it means that when moving the target element, it only processes the reference line for the graphical representation of the return value

Reply:
this is what I tried
   ```typescript
    initAligningGuidelines(this.canvas, {
            getObjectsByTarget: () => {
                const set = new Set<FabricObject>();
                for (const obj of this.canvas.getObjects()) {
                    if (obj.isGridLine) {
                        continue;
                    }
                    set.add(obj);
                }
                return set;
            },
        });
   ```
   I have a grid that I toggle, used as a helper when adding shapes 
   for aligning guidelines I want to ignore all lines in that grid
   and I thought I can ignore them if I don't add them to the set that is being returned

Reply:
@Smrtnyk 
I wrote the custom code according to your method and found that it works. You can compare it with the demo to see where the differences are.
[custom demo](https://codesandbox.io/p/devbox/fabric-aligning-guidelines-4pr6x7?file=%2Fsrc%2Ftest-custom.tsx%3A17%2C39)


https://github.com/user-attachments/assets/afac664a-d99a-4f03-ae7e-e77990f072c5